server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestRolloverIndexAction.java

```diff
@@ -51,6 +53,11 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         rolloverIndexRequest.lazy(request.paramAsBoolean("lazy", false));
         rolloverIndexRequest.timeout(request.paramAsTime("timeout", rolloverIndexRequest.timeout()));
         rolloverIndexRequest.masterNodeTimeout(request.paramAsTime("master_timeout", rolloverIndexRequest.masterNodeTimeout()));
+        if (DataStream.isFailureStoreEnabled()) {
+            rolloverIndexRequest.setFailureStoreOptions(
+                IndicesOptions.FailureStoreOptions.parseParameters(request.param("failure_store"), null)
```

Comment:
Since the rollover API does not support multi-target syntax, do we want to use the `failure_store` as a string with options `only` and `false` or should we use it as a boolean with `true` and `false`?

Reply:
Ah I wanted to put this in the PR description as a point of discussion, but forgot. Thanks for bringing this up :). I was on the fence about this but decided to do it this way to maintain consistency with the whole `IndicesOptions` system and to be able to reuse that system for parsing, passing, etc. I do agree that a boolean is a bit more straightforward (to understand), so I'm also good with switching to a boolean if others prefer that.

Reply:
Let's wait for someone else to chime in. I do see the point about consistency, but I do think it makes this API a bit harder to understand. So, I am slightly leaning towards a boolean `failure_store` param. Or potentially something completely different, `store=failure` or `store=backing`*? Just brainstorming here.

*PS: I do not fully like that we are introducing new terminology, because failure store is a term we have but backing store is not :| ....

Reply:
I'd be more in favor of the boolean version then. In the comments we do use the term `backing` a couple of times, I think, but not in APIs or features themselves indeed. +1 on letting someone else chime in

Reply:
Hmm, I checked in on some notes of mine from early designs and I think originally we had considered the idea of using `failure_store=include|exclude|only` for read operations and for write/mutation operations that need one or the other we went with `failure_store=true/false`. Really wish I had remembered that detail when reviewing the original flag changes. 😩